src/code-examples/accordion/html.js

```diff
@@ -1,25 +1,24 @@
-const html = `
-<div class='accordion'>
+const html = `<div class='accordion'>
   <div class='accordion-section'>
     <h3 class='accordion-header collapsed'>
-      <button id='sect1-button' type='button' role='button' aria-expanded='false' aria-controls='sect1-panel' type='button'>
+      <button id='sect1-button' type='button' aria-expanded='false' aria-controls='sect1-panel'>
         Section1
```

Comment:
Throughout this example code, it looks like I sometimes used single quotes instead of double quotes like the rest of our HTML examples. Would you mind switching those for me?

Reply:
Totally! I found a instance where I did that in the dialog component, so they both should be changed: https://github.com/sparkbox/accessible-components/pull/134/commits/fa754a38868323b6a937351b02432c3357865778